pkg/crypto/rings/client.go

```diff
@@ -53,15 +55,19 @@ func NewRingClient(deps depinject.Config) (_ crypto.RingClient, err error) {
 	return rc, nil
 }
 
-// GetRingForAddress returns the ring for the address provided.
+// GetRingForAddress returns the ring for the address and the block height provided.
+// The block height provided is used to determine the appropriate delegated gateways
+// to use at that height since signature verification may be performed for
+// delegations that are no longer active.
```

Comment:
1. I understand why you added these three sentence, but can you add a `TODO(@red-0ne): Link to docs once added`. Only you or I would understand what this means.
2. Is it worth mentioning we "round" the height to the latest session end?